qemu/tests/cfg/uefishell.cfg

```diff
@@ -81,6 +81,21 @@
             command_smbios = "smbiosview"
             variants:
                 - @default:
+                    check_result_smbios = "Version.*:\s+2.\d+"
+                - with_entry_point_type_32:
+                    no Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1
+                    smbios_output_handler = "handle_smbiosview"
+                    bios_version = "BiosVersion.*:.*edk2-([0-9]{8})"
+                    bios_release_date = "BiosReleaseDate.*:.*([0-9]{2})/([0-9]{2})/([0-9]{4})"
+                    machine_type_extra_params = "smbios-entry-point-type=32"
+                    smbios_version = "Version.*:\s+2.\d+"
+                - with_entry_point_type_64:
+                    no Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1
+                    smbios_output_handler = "handle_smbiosview"
+                    bios_version = "BiosVersion.*:.*edk2-([0-9]{8})"
+                    bios_release_date = "BiosReleaseDate.*:.*([0-9]{2})/([0-9]{2})/([0-9]{4})"
```

Comment:
p = "BiosReleaseDate.*:.*(?P<month>[0-9]{2})/(?P<day>[0-9]{2})/(?P<year>[0-9]{4})"
m = re.search(p, s)
m.group('month')
m.group('day')

Just FYI, yours is still OK for me

Reply:
updated, thanks.